tempest/api/workloadmgr/snapshot/test_snapshots.py

```diff
@@ -49,6 +49,7 @@ def test_1_create_full_snapshot(self):
             global volume_id
             global workload_id
             global snapshot_id
+            global full_snapshot_validations_before
```

Comment:
where is this global variable getting used ?

Reply:
It was used previously, removed as no longer needed.